src/depfile.rs

```diff
@@ -39,6 +39,12 @@ fn read_path<'a>(scanner: &mut Scanner<'a>) -> ParseResult<Option<&'a str>> {
                 scanner.back();
                 break;
             }
+            '\\' => {
+                if scanner.peek() == '\n' {
+                    scanner.back();
+                    break;
+                }
```

Comment:
I guess this means that backslashed escapes otherwise are treated as verbatim parts of paths, but I think you aren't changing this behavior, so it's something to fix later.